openshift/topics/advanced_concepts.adoc

```diff
@@ -281,6 +281,12 @@ If successful, the previous command will return the following output:
 route "sso" patched
 ----
 
+[[sso-connecting-to-an-external-database]]
```

Comment:
@slaskawi This should be added to the `Tutorials` section of the book, rather than to the `Advanced Concepts` one.

Reply:
@iankko Why? Is this a tutorial?

Reply:
As I explained in the [KEYCLOAK-10184](https://issues.jboss.org/browse/KEYCLOAK-10184), I would prefer to have a concise information about this topic. Unless you insist, I would like to keep it here (and not produce a longer tutorial).

Reply:
> @iankko Why? Is this a tutorial?

The `tutorials` subchapter would be a proper place in the case we would want to turn this into a tutorial (that was my original understanding). But since the real problem, why sharing an advanced link to official OCP documentation on how to configure external DB service isn't enough is [CLOUD-3198](https://issues.jboss.org/browse/CLOUD-3198), after the discussion / review, I am OK with providing a paragraph to the advanced section. The real issue is [CLOUD-3198](https://issues.jboss.org/browse/CLOUD-3198), and once that one is corrected, link to official documentation is enough.